packages/core/package.json

```diff
@@ -17,7 +17,7 @@
     "add-connector": "node build/cli/add-connector.js",
     "add-official-connectors": "node build/cli/add-official-connectors.js",
     "alteration": "node build/cli/alteration.js",
-    "test": "jest",
+    "test": "jest --testPathIgnorePatterns=/core/connectors/",
```

Comment:
this option can be added to `jest.config` file

Reply:
moved this option to config file